WSSiOS/WSSiOS/Source/Data/DTO/NovelDetailFeedResponse.swift

```diff
@@ -7,7 +7,7 @@
 
 import Foundation
 
-struct NovelDetailFeedResult: Codable {
+struct NovelDetailFeedResponse: Decodable {
     let isLoadable: Bool
-    let feeds: [TotalFeeds]
+    let feeds: [TotalFeedListDTO]
```

Comment:
```suggestion
    let feeds: [FeedListResponse]
```

W2;
해당 구조체가 NovelDetail, TotalFeed 둘 다 쓰이면 네이밍 수정하는 게 좋아보이네요~
저희 그리고 DTO 말고 Response로 네이밍 컨벤션 설정했던 것 같습니당